### DISCUSSION TITLE
part-ii/2020-49/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#50] [Question comments]
Part II 2020 Differential Geometry: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2020-49/

**Comment by stevezshen** [comment]
Should this be lim inf?

**Reply by eha1234** [reply]
I don't think so

**Comment by ArthurConmy** [comment]
Does something like this work for the final part? We have a point with $K \ge 1/R$. By the $|\partial_r K|$ condition, locally around this point, for a sufficiently small $\varepsilon $, $K \ge 1 / 2R$ and so $ \int K \ge \delta > 0 $. Then since the areas of $S_n$ are bounded above by a constant, on a subsequence $S_{n_i}$ with $ \text{inf}_{x \in S_{n_i}} K(x) \to 0 $, we will have $\int_{S_{n_i}} K \ge 0$ eventually, hence eventually all surfaces here will be $\cong S^2$.
